### ISSUE TITLE
Documenting Spatial Record Location

### ISSUE THREAD

**Issue by knelson-farmbeltnorth** [ADAPT/Standard#63]
Going forward we need to clarify expected usage of the location of Latitude/Longitude values reported on Spatial Record.   Some data will be reported from the GPS receiver, and some from the Implement Reference Point.   We need to add something so that the location is clearly defined from any given dataset.

**Comment by knelson-farmbeltnorth** [comment]
Initial proposal:   Add an EnumeratedValue called GeometryLocationType with initial values
-Machine GPS Receiver
-Implement GPS Receiver
-Implement Reference Point

**Comment by zwing99** [comment]
Can we make sure this gets attached appropriately within the model and not just added everywhere and left optional but the idea is sound!

**Comment by knelson-farmbeltnorth** [comment]
OperationData is the correct place to put this.    It already contains the List of Spatial Records and an optional property for their count.   

Agreed in principle at the Sept 7 2022 meeting subject to the constraints mentioned by @zwing99

**Comment by knelson-farmbeltnorth** [comment]
When implementing this, I identified the need to add a 4th option to accommodate polygon data for which a specific lat/lon does not pertain as the coverage geometry is explicitly defined in the Spatial Record vs. requiring consumers to derive it.   I defined the property thus:

Defines the location of the geometry property as reported on the Spatial Records.  One of:  
Machine GPS Receiver
Implement GPS Receiver
Implement Reference Point
Predefined Polygon

@strhea @zwing99
